crates/sqlez_macros/src/sqlez_macros.rs

```diff
@@ -1,12 +1,16 @@
 use proc_macro::{Delimiter, Span, TokenStream, TokenTree};
-use sqlez::thread_safe_connection::{locking_queue, ThreadSafeConnection};
 use syn::Error;
 
-lazy_static::lazy_static! {
-    static ref SQLITE: ThreadSafeConnection =  {
-        ThreadSafeConnection::new(":memory:", false, None, Some(locking_queue()))
-    };
-}
+#[cfg(not(target_os = "linux"))]
```

Comment:
Curious why we needed to add this target check here, since it wasn't there before?

Reply:
I think the conditional check is necessary since we only use for SQLITE there for linux.

Reply:
But why wasn't it there before?

If it's not directly related to the `LazyLock` changes I would prefer it did not go in this PR.

Reply:
Alright, I figured out why we need this now:

`lazy_static!` automatically marks the generated static with `#[allow(dead_code)]`: https://github.com/rust-lang-nursery/lazy-static.rs/blob/cc81425f478f20eb286b3698ade99a99dd41a084/src/lib.rs#L150